spec/requests/support_interface/not_authorized_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'GET /support/applications' do
+  context 'when the DfE Sign-in account is not authorized' do
+    before do
+      allow(SupportUser).to receive(:load_from_session)
+        .and_return(
+          SupportUser.new(
+            email_address: 'email@example.com',
+            dfe_sign_in_uid: 'DFE_SIGN_IN_UID',
+          ),
+      )
+
+      # do not grant the user permission to view a provider's applications
```

Comment:
Would it be worth clarifying this comment to say that the permission here is to view/use support?